lib/matplotlib/_pylab_helpers.py

```diff
@@ -58,27 +55,16 @@ def destroy(cls, num):
             return
         manager = cls.figs[num]
         manager.canvas.mpl_disconnect(manager._cidgcf)
-
-        # There must be a good reason for the following careful
-        # rebuilding of the activeQue; what is it?
-        oldQue = cls._activeQue[:]
-        cls._activeQue = []
-        for f in oldQue:
-            if f != manager:
-                cls._activeQue.append(f)
-
+        cls._activeQue.remove(manager)
```

Comment:
The functional difference here is that the old code *replaces* the class attribute `_activeQue`, while the new code modifies the original.  The only way I can see this would make a difference would be if there were other references to _activeQue.  It doesn't look like that should occur; and if it did, I don't see how the replacement would be helpful.  Therefore, I think this is OK.